src/renderers/shared/ReactDebugTool.js

```diff
@@ -68,7 +69,7 @@ var ReactDebugTool = ((null: any): typeof ReactDebugTool);
 if (__DEV__) {
   const hooks = [];
   const didHookThrowForEvent = {};
-  const ReactComponentTreeHook = require('react/lib/ReactComponentTreeHook');
+  const ReactComponentTreeHook: ComponentTreeHookDevType = (require('react/lib/ReactComponentTreeHook'): any);
```

Comment:
@gaearon @bvaughn It seems like we do want to check for each function before calling it so that react-dom.js works with react.min.js, unless you don't think that's valuable.